Makefile.in

```diff
@@ -946,10 +954,12 @@ stamps/build-llvm-linux: $(LLVM_SRCDIR) $(LLVM_SRC_GIT) \
 	    -DDEFAULT_SYSROOT="../sysroot" \
 	    -DLLVM_RUNTIME_TARGETS=$(call make_tuple,$(XLEN),linux-gnu) \
 	    -DLLVM_INSTALL_TOOLCHAIN_ONLY=On \
+	    -DLLVM_BINUTILS_INCDIR=`find $(INSTALL_DIR) -name plugin-api.h | xargs dirname` \
```

Comment:
Maybe just `-DLLVM_BINUTILS_INCDIR=$(BINUTILS_SRCDIR)/include`? and don't forgot add `$(BINUTILS_SRCDIR) $(BINUTILS_SRC_GIT)` to `stamps/build-llvm-newlib` and `stamps/build-llvm-linux`